src/content/nl/2020/mobile-web.md

```diff
@@ -0,0 +1,417 @@
+
+Een <a hreflang="en" href="https://web.dev/milliseconds-make-millions/">recente studie</a> toonde aan dat 0,1s mSpeed verbetering de conversieratio's met +8,4% verhoogde voor winkelsites en +10,1% voor reissites.
+
+Het laden van prestaties is een veelomvattend onderwerp, dus hebben we een aantal aspecten uitgekozen om hier te behandelen.
+
+##### Largest Contentful Paint
+
+Een belangrijk aspect van de laadervaring is hoe snel de hoofdinhoud van een webpagina wordt geladen en zichtbaar is voor gebruikers. Dit was moeilijk te meten, in het verleden heeft Google prestatiestatistieken aanbevolen, zoals <a hreflang="en" href="https://web.dev/first-meaningful-paint/">First Meaningful Paint</a> (FMP) om dit vast te leggen, maar het was moeilijk uit te leggen en vaak niet in staat om vast te stellen wanneer de hoofdinhoud van de pagina zichtbaar was.
+
+Soms is eenvoudiger beter. Meer recentelijk is ontdekt dat een nauwkeurigere manier om te meten wanneer de hoofdinhoud van een pagina is geladen, is door simpelweg te kijken naar wanneer het grootste element is weergegeven. <a hreflang="en" href="https://web.dev/largest-contentful-paint">Largest Contentful Paint</a> (LCP) is een op timing gebaseerde statistiek die dit vastlegt: het tijdstip waarop de grootste element boven de vouw is weergegeven.
+
+Een goede LCP-score is 2,5s op p75. We ontdekten dat het gemiddelde LCP op p75 2,6s op mobiel en 2,3s op desktop is. Vooral mobiel internet mist het doel op LCP.
+
+{{ figure_markup(
+  image="median-p75-lcp-score.png",
+  alt="Mediane p75 LCP-score",
+  caption='Mediane p75 LCP-score',
+  description="Een grafiek die laat zien dat het gemiddelde LCP op p75 2,6s is op mobiel en 2,3s op desktop",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQNyVui4x9yanLKpp8Zz92IJ9c2NKCN_2g7SH-vRoelqT3nABB_uTQnXXScCUo6D-Uis1_wDzwcdGzx/pubchart?oid=964425991&format=interactive",
+  sheets_gid="872081120"
+  )
+}}
+
+##### Afbeeldingen
+
+Hoewel elk type item, zoals lettertype, CSS, JavaScript, enz. een belangrijke rol speelt bij de laadprestaties, nemen we afbeeldingen onder de loep.
+
+Het internet blijft evolueren naar pagina's met veel afbeeldingen, met de groei van bandbreedte en de alomtegenwoordigheid van smartphones. En afbeeldingen brengen kosten met zich mee voor de laadprestaties.
+
+Afbeeldingen met een onjuist formaat en niet-geoptimaliseerde afbeeldingen zijn frequente bronnen voor problemen met de afbeeldingsprestaties. Maar liefst 41,20% van de pagina's heeft afbeeldingen met een onjuist formaat.
+
+{{ figure_markup(
+  image="pages-with-properly-sized-images.png",
+  alt="Pagina's met afbeeldingen van het juiste formaat",
+  caption='Pagina's met afbeeldingen van het juiste formaat',
```

Comment:
Need to change caption to double quotes. And I can't see any reason for the extra `alt` so can we remove that (in English too if possible):

```suggestion
  caption="Pagina's met afbeeldingen van het juiste formaat",
```